packages/next-on-pages/templates/_worker.js/routes-matcher.ts

```diff
@@ -361,6 +361,9 @@ export class RoutesMatcher {
 			this.path = this.path.replace(/\.rsc/i, '');
 		}
 
+		// Prevent `//` in path names.
+		this.path = this.path.replace(/\/\//g, '/')
```

Comment:
I think that this is ok as `//` is always ok to squash to `/` 🤔 

but I think it'd be great to investigate this further and understanding why we end up with `//` 🤔 

I'm guessing that we're doing something wrong earlier during the request handling process

I'm happy to investigate this and or @simmbiote please let me know if you'd like to 🙂